scripts/uosc.lua

```diff
@@ -4280,7 +4281,11 @@ mp.observe_property('playlist-count', 'number', function(_, value)
 end)
 mp.observe_property('fullscreen', 'bool', create_state_setter('fullscreen', update_fullormaxed))
 mp.observe_property('window-maximized', 'bool', create_state_setter('maximized', update_fullormaxed))
-mp.observe_property('idle-active', 'bool', create_state_setter('idle'))
+mp.observe_property('idle-active', 'bool', function(_, idle)
+	set_state('is_idle', idle)
+	create_state_setter('idle')
```

Comment:
Remove the `create_state_setter('idle')` call. It doesn't do anything here.

Reply:
Could simply replace `'idle'` with `'is_idle'` in the original code and that should be fine or not?

Reply:
Depends on the order in which these events are being dispatched. If `idle-active` is last after everything else that triggers `dispositions` than this won't be reliable. I guess better be safe.

Reply:
Oh right, I missed the disposition because I only looked at that excerpt.

Reply:
> Remove the `create_state_setter('idle')` call. It doesn't do anything here.

Done.